source/standards/storing-credentials.html.md.erb

```diff
@@ -0,0 +1,60 @@
+---
+title: How to store credentials
+last_reviewed_on: 2021-08-06
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+Depending on how you [manage your accounts](/standards/accounts-with-third-parties.html), you, your team and the service you run may have credentials or other secrets that you need to store securely.
+
+## Personal credentials
+
+Examples include login details for:
+
+- GitHub
+- AWS
+- GOV.UK Signon
+
+[Use the password manager built into your browser in the first instance](https://lock.cmpxchg8b.com/passmgrs.html#conclusion). This is simpler than setting up an extra account with a third party and avoids some of the potential issues below.
+
+If you can't do this for some reason, then it's OK to use a third-party password manager. This could be necessary if your browser has an accessibility issue, or if you work with multiple browsers.
+
+Some of the third-party password managers people use at GDS include:
```

Comment:
@risicle just following up on [your comment about QtPass](https://github.com/alphagov/gds-way/pull/633#issuecomment-897481484).

I had a quick look and it seems to be an offline, GPG-based approach. Do you think we should recommend other people use it here?

Reply:
Well, I'd recommend it as much as any of the others.

Reply:
(It also has git support to let you push your stores around)

Reply:
Haha, OK. It sounds like an interesting choice and good to have one other offline option for balance (without turning this into a list of all password managers!).

I've added it in a new commit: https://github.com/alphagov/gds-way/pull/633/commits/4b53bb13befd0f7059791dc41157ccda5977b068